src/service.rs

```diff
@@ -479,13 +481,15 @@ impl<P: Platform> ServiceResources<P> {
             },
 
             Request::RequestUserConsent(request) => {
-                assert_eq!(request.level, consent::Level::Normal);
+                // assert_eq!(request.level, consent::Level::Normal);
```

Comment:
Why comment this out?

Reply:
There is little point in panicking in production code. If anything, this should be a debug_assert. Also we handle the Strong case too. Or do you mean I should have deleted the line completely? :)

Reply:
I meant you should delete the line completely. There isn't much use in having commented out code